src/mscorlib/src/System/Reflection/MethodBase.Unsharable.cs

```diff
@@ -0,0 +1,161 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
'kay - I figured not hardcoding the platform was more flexible but since there's prior precedent...